build-system/tasks/extension-helpers.js

```diff
@@ -91,25 +91,17 @@ const DEFAULT_EXTENSION_SET = ['amp-loader', 'amp-auto-lightbox'];
  *   cssBinaries?: Array<string>,
  *   extraGlobs?: Array<string>,
  *   binaries?: Array<ExtensionBinary>,
- *   npm?: Map<string, NpmBinaries>,
+ *   npm?: boolean,
  * }}
  */
 const ExtensionOption = {}; // eslint-disable-line no-unused-vars
```

Comment:
Pretty sure you can make this `ExtensionOptionDef` and remove the lint override.

Reply:
Great point!